src/Services/FHIR/Observation/FhirObservationVitalsService.php

```diff
@@ -49,7 +49,7 @@ class FhirObservationVitalsService extends FhirServiceBase implements IPatientCo
 
     const CATEGORY = "vital-signs";
 
-    private const COLUMN_MAPPINGS = [
+    const COLUMN_MAPPINGS = [
```

Comment:
The only downside about this, is that I wasn't sure I wanted to expose all of this data for external use of the class as some of the internal implementations may change (which is what I'm doing now).  Maybe we need another constant that has just the codes... I think this is fine for now, I may change this in the work I'm doing once I rebase on what you've put in.